### DISCUSSION TITLE
Hidden Item Folder/Album - Option for PIN protect

### DISCUSSION THREAD

**Discussion by s38b35M5** [ente-io/ente#636] [Enhancements]
(This issue is perhaps best-suited to the desktop app, as mobile can simply use biometric auth as needed)
When you want to access your Hidden items, you are prompted to enter your account password, which enhances security considerably. However, in my case, my account password is complex and long, and only known to my password manager. 

If there was an option to use a PIN or non-account Password/phrase, that could enhance usability while still providing security. It could default to the current arrangement, and have a setting to use less-secure authentication to provide access.

[Priority: Low]

**Comment by s38b35M5** [comment]
Placeholder comment to request clarification of this issue being reduced to a discussion

**Reply by mnvr** [reply]
The word "discussion" doesn't mean we're considering it less important, we have just migrated feature requests to discussions. When migrating, GitHub apparently closes the original.

The feature request is still not closed, if that is what your concern was. Our hope is that since in the Discussions the upvotes are shown at the top level by GitHub, we can keep our feature requests here and use the community upvotes to get some sense of which features to pick first. More information is here - https://github.com/ente-io/ente/discussions/480. 

Not just your issue, but other issues too are being moved (this is related to us open sourcing our servers yesterday)

**Reply by s38b35M5** [reply]
Thank you for the explanation. However, keep in mind the current app points the user to the issues page when clicking "Feature Request."

**Reply by mnvr** [reply]
Yes, I know. I'm trying to put up a banner to redirect to here (And of course, we'll fix the links in the apps in the next update)

**Comment by geronimoge** [comment]
I would love to see this feature in the future in the mobile app besides biometric authentication.

**Comment by Ravenguard77** [comment]
Sounds like a good option
